src/pages/NewsList/NewsList.tsx

```diff
@@ -85,54 +78,91 @@ export default function NewsList() {
 		data = [];
 	}
 
+	let filteredData = data;
+
+	if (searchText) {
+		filteredData = data.filter(
+			(newsItem: NewsData) => newsItem.title.toLowerCase().includes(searchText.toLowerCase())
```

Comment:
you can do `searchText.toLowerCase()` only once and then use it. And also there are probably more ways to simplify this. Right now it is not easy to read.